### PR TITLE
docs: #26 add conventions and project structure

### PR DESCRIPTION
**Pull request by Ekaterina1994**

### CODE REVIEW

File: mw-webapp/.eslintrc [hunk 1]
```diff
@@ -86,6 +86,8 @@
     "curly": "error",
     "comma-dangle": ["error", "always-multiline"],
     "brace-style": "error",
+    // "no-console": ["error"],
```
Comment (tritonJS826) [inline mw-webapp/.eslintrc:89]: comment

**Review by tritonJS826 — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -2,6 +2,13 @@
 
 Multi-user application for tracking the educational process.
 
+### Technologies
+
+- React
+- TypeScript
+- Vite
+- SCSS modules
+
```
Comment (SergioAJS) [inline README.md:11]: Maybe add Redux. We planned to use it

**Review by SergioAJS — APPROVED** [review]

File: mw-webapp/.eslintrc (hunk 1 above)
Comment (Ekaterina1994) [reply mw-webapp/.eslintrc]: Delete

File: README.md (hunk 2 above)
Comment (Ekaterina1994) [reply README.md:11]: But we don't use it now. We can add Redux when will use it)

Comment (SergioAJS) [reply README.md:11]: Ok

**Pull request merged**